README.adoc

```diff
@@ -130,7 +130,7 @@ Create a project by running the following command:
 
 [role=command]
 ```
-oc new-project [project-name]
+oc new-project guide
```

Comment:
@AustinSeto  any reason change to `guide`?

Reply:
Reverted with mentions to replace `[project-name]` with name of user choice